### PR TITLE
Add debug ping view how-to article 

### PR DESCRIPTION
**Pull request by georgf**
- Moved debug ping view how-to into article here.
- Moved glean into it's own section, with the new article a sub-article.

### CODE REVIEW

File: src/concepts/glean/debug_ping_view.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  --ez logPings true \
+  --es sendPing baseline \
+  --es tagPings my-debug-tag
+```
+
+_my-debug-tag_ is what will help you identify your data in the web interface, while `<application-id>`  is the application identifier as declared in the [manifest](https://developer.android.com/studio/build/application-id) (e.g. `org.mozilla.reference.browser`). The debug commands are documented in more detail [in the Glean README](https://github.com/mozilla-mobile/android-components/blob/master/components/service/glean/README.md#debugging-products-using-glean).
+
+### Supported applications
+
+As for now, the following application ids are supported:
```
Comment (akkomar) [inline src/concepts/glean/debug_ping_view.md]: We currently have part of the schema aliasing mechanism deployed. Although it doesn't pick up schemas and applications automatically yet, we can manually update config file to add support for more applications.

File: src/concepts/glean/debug_ping_view.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+*   `org.mozilla.fenix`
+*   `org.mozilla.reference.browser`
```
Comment (Dexterp37) [inline src/concepts/glean/debug_ping_view.md]: @akkomar , do we also support [firefox-tv](https://github.com/mozilla-mobile/firefox-tv/blob/1416b41b68f93adbf94803c4b9ae45d2b4330a8c/app/build.gradle#L23)?

File: src/concepts/glean/debug_ping_view.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+## Where can I see the data?
+
+The data is provided in [this web interface](https://debug-ping-preview.firebaseapp.com/). It lists all recently active devices and updates automatically. You can use your debug identifier to quickly identify your own testing data.
+
+Any data sent from a mobile device usually shows up within 10 seconds, updating the pages automatically.
+
+## Can you give me an example?
+
+For example to send a baseline ping immediately from the reference browser, with a debug identifier of `johndoe-test1`:
```
Comment (Dexterp37) [inline src/concepts/glean/debug_ping_view.md]: nit: should we use capital initials for Reference Browser?

File: src/concepts/glean/debug_ping_view.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+*   `org.mozilla.reference.browser`
+*   `org.mozilla.samples.glean`
+
+## Where can I see the data?
+
+The data is provided in [this web interface](https://debug-ping-preview.firebaseapp.com/). It lists all recently active devices and updates automatically. You can use your debug identifier to quickly identify your own testing data.
+
+Any data sent from a mobile device usually shows up within 10 seconds, updating the pages automatically.
+
+## Can you give me an example?
```
Comment (Dexterp37) [inline src/concepts/glean/debug_ping_view.md]: Any chance we could link or embed [this](https://github.com/mozilla-mobile/android-components/tree/master/components/service/glean#important-gleandebugactivity-notes) from the Glean SDK docs? We had a couple of bugs filed that made us notice that the supported flows were not clear :(

File: src/concepts/glean/debug_ping_view.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+**Note**: Glean will always attempt to collect data for the ping that was requested using the `sendPing` command line switch. However, if no data is recorded by the application, nothing will be sent. The `baseline` ping is _guaranteed_ to always be sent, since it’s populated by Glean itself.
+
+
+## Troubleshooting
+
+If nothing is showing up on the dashboard, it would be useful to check the following:
+
+*   If `adb logcat` reports _”Glean must be enabled before sending pings.”_ right after calling the `GleanDebugActivity`, then the application has disabled Glean. Please check with the application team on how to fix that.
+*   If no error is reported when triggering tagged pings, but the data won't show up on the dashboard, check if the used `<application-id>` is the same expected by the Glean pipeline (i.e. the one used to publish the application on the Play Store).
+*   Before April 12 builds, Fenix had a crashing bug around Glean. This is now fixed, please update.
```
Comment (Dexterp37) [inline src/concepts/glean/debug_ping_view.md]: Is this still useful?

File: src/concepts/glean/debug_ping_view.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+*   The reference-browser debug builds currently don't enable Glean. You could override this in local builds.
```
Comment (Dexterp37) [inline src/concepts/glean/debug_ping_view.md]: Fenix has the same behaviour.

File: src/concepts/glean/debug_ping_view.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
+# Using the Glean debug ping view
+
+<!-- toc -->
+
+## What is this good for?
+
+This enables you to easily see in real-time what data your mobile application is sending through [Glean](glean.md).
```
Comment (badboy) [inline src/concepts/glean/debug_ping_view.md]: IMO this should start with the name of `this`:

```suggestion
Glean Debug Ping View enables you to easily see in real-time what data your mobile application is sending through [Glean](glean.md).
```

File: src/concepts/glean/debug_ping_view.md (hunk 5 above)
Comment (badboy) [inline src/concepts/glean/debug_ping_view.md]: Would it make sense to already add a year here?

File: src/concepts/glean/debug_ping_view.md (hunk 2 above)
Comment (akkomar) [reply src/concepts/glean/debug_ping_view.md]: Yes, we support: Fenix, &browser, samples-glean, tv-firefox and some debug flavors.

Comment (akkomar) [reply src/concepts/glean/debug_ping_view.md]: Note to self: we should remove this section after automated process for managing schemas is in place.

File: src/concepts/glean/debug_ping_view.md (hunk 5 above)
Comment (georgf) [reply src/concepts/glean/debug_ping_view.md]: Or we just remove it, it shouldn't affect current users anymore.

Comment (georgf) [reply src/concepts/glean/debug_ping_view.md]: Yeah, probably not.

**Pull request merged**